packages/transition-backend/src/services/executableJob/ExecutableJob.ts

```diff
@@ -40,7 +40,7 @@ export class ExecutableJob<TData extends JobDataType> extends Job<TData> {
             console.log(`Enqueuing ${runningAndPending.jobs.length} in progress and pending jobs`);
             for (let i = 0; i < runningAndPending.jobs.length; i++) {
                 const job = new ExecutableJob<TData>(runningAndPending.jobs[i] as JobAttributes<TData>);
-                await job.enqueue(progressEmitter);
+                job.enqueue(progressEmitter);
             }
 
             return true;
```

Comment:
What does the return true here means?
DOes the mean clash with the fact that we don't await the enqueuing?

Reply:
That there was no exception and everything was correct, instead of just returning nothing... we don't do anything with it anyway